cards/src/main/resources/cards/custom/group3/bug/verdant_dreams/spell_string_shot.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "String Shot",
+  "baseManaCost": 1,
+  "type": "SPELL",
+  "heroClass": "AMBER",
+  "rarity": "RARE",
+  "description": "If you control a Beast, return an enemy minion to your opponent's hand.",
+  "targetSelection": "ENEMY_MINIONS",
+  "condition": {
+    "class": "AnyMatchFilterCondition",
+    "target": "FRIENDLY_MINIONS",
+    "filter": {
+      "class": "RaceFilter",
+      "race": "BEAST"
+    }
+  },
+  "spell": {
+    "class": "ReturnTargetToHandSpell"
```

Comment:
i forget if you need to specify target player here? whatever sap does

Reply:
This is what sap does.

Reply:
```
{
  "name": "Sap",
  "baseManaCost": 2,
  "type": "SPELL",
  "heroClass": "BLACK",
  "rarity": "FREE",
  "description": "Return an enemy minion to your opponent's hand.",
  "targetSelection": "ENEMY_MINIONS",
  "spell": {
    "class": "ReturnTargetToHandSpell"
  },
  "attributes": {
    "SPELLSOURCE_NAME": "Paralyzing Touch"
  },
  "collectible": true,
  "set": "BASIC",
  "fileFormatVersion": 1
}
```